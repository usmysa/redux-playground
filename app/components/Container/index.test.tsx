import { renderWithProviders } from "@/__tests__";
import { Container } from ".";

describe("Container", () => {
  it("renders correctly", () => {
    const { container } = renderWithProviders(
      <Container>
        <div>Container</div>
      </Container>,
    );

    expect(container).toMatchSnapshot();
  });
});
