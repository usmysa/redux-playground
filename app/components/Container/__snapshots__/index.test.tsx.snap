// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Container > renders correctly 1`] = `
<div>
  <div
    class="styles_wrapper__utfcpz0"
  >
    <div>
      Container
    </div>
  </div>
</div>
`;
